README.md

```diff
@@ -10,8 +10,8 @@ See CONTRIBUTING.md for more details on setting up.
 
 -->
 
-WARNING: This is the master branch. The current release v1.0.0-beta2
-can be found [here](https://github.com/elastic/ecs/tree/v1.0.0-beta2).
+WARNING: This is the master branch. The current release v1.0.0
```

Comment:
I assume this should be removed from the branch?

Reply:
Well the readme as we know it will be gone soon, with the docs moving to the main site. So let's not sweat it too much. But I'll make this read "this is a development branch".